adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -124,8 +124,6 @@ public SwitchToBroker canSwitchToBroker(final String authorityUrlStr) {
         }
         final String packageName = mContext.getPackageName();
         boolean canSwitchToBroker = AuthenticationSettings.INSTANCE.getUseBroker()
-                && !packageName.equalsIgnoreCase(AuthenticationSettings.INSTANCE.getBrokerPackageName())
```

Comment:
We'll need to get Company Portal to test out these changes too, but can merge first and be reintroduced/modified later as needed

Reply:
Added a comment on top of the function as a reminder.